api/v1/openapi.json

```diff
@@ -0,0 +1,1115 @@
+                                                }
+                                        }
+				}
+			},
+			"delete": {
+				"summary": "Delete a service",
+				"tags": [ "service" ],
+				"parameters": [
+					{ "$ref": "#/parameters/service-id" },
+					{ "$ref": "#/parameters/service-address" }
+				],
+				"responses": {
+					"200": {
+                                                "description": "Success"
+					},
+                                        "404": {
+                                                "description": "Service not found",
+                                                "schema": {
+                                                        "$ref": "#/definitions/Error"
+                                                }
+                                        }
+				}
+			}
+		}
+        },
+	"parameters": {
+		"endpoint-id": {
+			"name": "id",
+			"description": "Endpoint ID",
+			"in": "query",
+			"required": true,
+			"type": "string"
+		},
+		"endpoint-id-type": {
+			"name": "type",
+			"description": "Type of endpoint ID",
+			"in": "query",
+			"type": "string",
+			"default": "cilium",
+			"enum": [
+				"cilium",
+				"docker-id",
+				"docker-endpoint"
+			]
+		},
+		"identity-id": {
+			"name": "id",
+			"description": "Identity ID",
+			"in": "query",
+			"type": "integer"
+		},
+		"identity-hash": {
+			"name": "hash",
+			"description": "SHA256 hash of identity labels",
+			"in": "body",
+			"schema": {
+				"type": "string"
+			}
+		},
+		"policy-path": {
+			"name": "path",
+			"description": "Path to policy node",
+			"in": "query",
+			"type": "string"
+		},
+		"policy-tree": {
+			"name": "policy",
+			"description": "Policy tree or subtree",
+			"required": true,
+			"in": "body",
+			"schema": {
+				"type": "string"
+			}
+		},
+		"identity-context": {
+			"name": "identity-context",
+			"description": "Context to provide policy evaluation on",
+			"in": "body",
+			"schema": {
+				"$ref": "#/definitions/IdentityContext"
+			}
+		},
+		"service-id": {
+			"name": "id",
+			"description": "ID of service",
+			"required": true,
+			"in": "path",
+			"type": "integer"
+		},
+		"service-address": {
+			"name": "address",
+			"description": "Service address configuration",
+			"in": "body",
+			"schema": {
+				"$ref": "#/definitions/FrontendAddress"
+			}
+		},
+		"service-config": {
+			"name": "config",
+			"description": "Service configuration",
+			"in": "body",
+			"schema": {
+				"$ref": "#/definitions/Service"
+			}
+		},
+		"ipam-ip": {
+			"name": "ip",
+			"description": "IP address",
+			"in": "path",
+			"required": true,
+			"type": "string"
+		},
+		"ipam-family": {
+			"name": "family",
+			"in": "query",
+			"type": "string",
+			"enum": [
+				"ipv4",
+				"ipv6"
+			]
+		}
+	},
+        "definitions": {
+		"Endpoint": {
+			"description": "Endpoint",
+                        "type": "object",
+			"properties": {
```

Comment:
This is represented by the `/endpoint/config/`. The question is whether this in fact needs to be a separate API path. It's separate right now to distinguish between create/delete/state-change of endpoints and mere configuration changes.